Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -1030,6 +1030,9 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         case .removeVPNConfiguration:
             // Since the VPN configuration is being removed we may as well reset all state
             handleResetAllState(completionHandler: completionHandler)
+        case .uninstallVPN:
```

Comment:
New command to avoid having to make two IPC calls from the client app to uninstall.